conf/pre_deploy_actions.bash

```diff
@@ -10,27 +10,24 @@ cd "$(dirname $BASH_SOURCE)"/..
 # them back to the defaults which is what they would have on the servers.
 PYTHONDONTWRITEBYTECODE=""
 
-# create the virtual environment; we always want system packages
-virtualenv_args="--system-site-packages"
+# create the virtual environment
 virtualenv_dir='.venv'
 virtualenv_activate="$virtualenv_dir/bin/activate"
 
 if [ ! -f "$virtualenv_activate" ]
 then
-    virtualenv $virtualenv_args $virtualenv_dir
+    python3 -m venv $virtualenv_dir
 fi
 
 source $virtualenv_activate
 
-# Upgrade pip to a secure version
-# curl -L -s https://raw.github.com/pypa/pip/master/contrib/get-pip.py | python
-# Revert to the line above once we can get a newer setuptools from Debian, or
-# pip ceases to need such a recent one.
-curl -L -s https://raw.github.com/mysociety/commonlib/master/bin/get_pip.bash | bash
-# Improve SSL behaviour
-pip install 'pyOpenSSL>=0.14'
+# Install GDAL, correct version, right headers
+gdal_version=$(gdal-config --version)
+# stretch version of GDAL doesn't have matching pypi version
+if [ $gdal_version = "2.1.2" ]; then gdal_version="2.1.3"; fi
```

Comment:
Think we said we could drop this line now.

Reply:
Ah yes, it's only needed on older operating systems - we'd only need it if we ever want to deploy this PR to Debian Stretch. I'll take that out.